Makefile

```diff
@@ -75,3 +75,16 @@ docker-php-apache:
 docker-push-php-apache:
 	eval $$(aws ecr get-login --registry-ids $(AKT_AWS_ACCOUNT_ID) --no-include-email --region $(AKT_AWS_REGION))
 	docker push $(AKT_AWS_ACCOUNT_ID).dkr.ecr.$(AKT_AWS_REGION).$(AKT_ECR_HOST)/$(PHP_APACHE_IMG_NAME):$(PHP_APACHE_TAG);
+
+CL2_AWS_ACCOUNT_ID ?= 197575167141
```

Comment:
You should be able to re-use the AKT_AWS_ACCOUNT_ID, it does nice project auto-discovery which enables things like:
AWS_PROFILE=scalability make docker-release.